src/main/java/randoop/main/GenInputsAbstract.java

```diff
@@ -1148,6 +1170,117 @@ private static boolean shouldOmitClass(String classname) {
     }
   }
 
+  /**
+   * Returns classes in the given package. Does not include classes in sub-packages. Ignores
+   * non-accessible classes.
+   *
+   * @param packageName a package name
+   * @param accessibility the accessibility predicate
+   * @return classes in package {@code packageName}
+   */
+  private static Set<@ClassGetName String> getClassnamesFromPackage(
+      String packageName, AccessibilityPredicate accessibility) {
+    Set<@ClassGetName String> classnames = new HashSet<>();
+
+    for (String path : Globals.getClassPath().split(File.pathSeparator)) {
+      File location = new File(path);
+      if (location.isFile() && location.getName().endsWith(".jar")) {
+        classnames.addAll(getClassesWithPackageFromJar(location, packageName, accessibility));
+      } else if (location.isDirectory()) {
+        classnames.addAll(getClassesWithPackageFromDirectory(location, packageName, accessibility));
+      }
+    }
+
+    return classnames;
+  }
+
+  /**
+   * Searches in the given directory for classes with the given package. Directory must be root
```

Comment:
I don't understand what a "root directory" is.  In typical file system parlance, this would be "/" in Unix or "C:\" in Windows.
Also, this method does *not* search the given directory.  It searches one subdirectory of the given directory, where the subdirectory corresponds to the package name.  Could you please clarify the comment?

I was confused before about whether the search is deep or shallow (that is, examines subdirectories or only files directly present in the given directory), but I think clarifying the documentation will prevent that confusion.